frontend/src/features/moti/mapSideBar/forms/PropertyForm.tsx

```diff
@@ -0,0 +1,22 @@
+import { Formik } from 'formik';
+import { noop } from 'lodash';
+import * as React from 'react';
+import { Form } from 'react-bootstrap';
+
+interface IPropertyFormProps {
+  formikRef: any;
```

Comment:
Do you have to pass the ref?  Is there a way to extract it from the hook?

Reply:
The formikRef is coming from the MotiSideBarContainer, not a hook. Remember that hooks don't preserve state between multiple invocations, so hooks aren't an effective way of sharing state between components.

What you may be thinking of is the pattern of creating a context with a hook that manages access to the context. That is certainly an option, but I guess my question is, why do you not want to pass the ref? Passing refs from parents to children is also a react pattern to allow parent->child communication.